layers/gd/Dockerfile

```diff
@@ -0,0 +1,55 @@
+ARG PHP_VERSION
+FROM bref/build-php-$PHP_VERSION as ext
+
+# Install required build libraries
+RUN set -ue \
+    ; LD_LIBRARY_PATH= \
+    ; yum -y install \
+        libwebp-devel \
+        libXpm-devel \
+        libpng-devel \
+        libjpeg-devel \
+        freetype-devel \
+    ; yum clean all \
+;
+
+WORKDIR ${PHP_BUILD_DIR}/ext/gd
+RUN phpize
+ARG PHP_VERSION
+RUN if [ "$PHP_VERSION" = "73" ] ; \
+    then \
+        ./configure \
+        --disable-static \
+        --with-gd \
+        --enable-gd-jis-conv \
+        --enable-shared \
+        --with-zlib-dir=${INSTALL_DIR} \
+        --with-jpeg-dir=${INSTALL_DIR} \
+        --with-png-dir=${INSTALL_DIR} \
+        --with-webp-dir=${INSTALL_DIR} \
+        --with-xpm-dir=${INSTALL_DIR} \
```

Comment:
please keep the same ordering in both parts, thus minimising errors when making a comparison (Alphabetical ordering would be the best IMHO)